daml-foundations/daml-ghc/daml-stdlib-src/DA/Text.daml

```diff
@@ -300,13 +300,15 @@ legacyParsePositiveDecimal cs =
   legacyParseIntToDecimal cs = legacyParseInt_ intToDecimal cs 0.0
 
 -- | Attempt to parse a `Decimal` value from a given `Text`.
--- To get `Some` value, the text must follow the regex '-'?[0-9]+('.'[0-9]+)?
--- In particular, the shorthands `".12"` and `"12."` do not work.
+-- To get `Some` value, the text must follow the regex
+-- `(-|\+)?[0-9]+('.'[0-9]+)?`
+-- In particular, the shorthands `".12"` and `"12."` do not work,
+-- but the value can be prefixed with `+`.
 -- Leading and trailing zeros are fine, however spaces are not.
 -- Examples:
 -- ```
 --   parseDecimal "3.14" == Some 3.14
---   parseDecimal "+12.0" == None
+--   parseDecimal "+12.0" == Some 12
```

Comment:
```suggestion
--   parseDecimal "+12.0" == Some 12.0
```
The `.0` is crucial for the number to be a `Decimal`. Otherwise it would be an `Int`.